### DISCUSSION TITLE
New Gatsby plugin for Decap is now available

### DISCUSSION THREAD

**Discussion by martinjagodic** [decaporg/decap-cms#7089] [Announcements]
The deprecated [gatsby-plugin-netlify-cms](https://www.npmjs.com/package/gatsby-plugin-netlify-cms) that corresponds to the old Netlify CMS package is now deprecated and replaced by [gatsby-plugin-decap-cms](https://www.npmjs.com/package/gatsby-plugin-decap-cms).

To migrate you Gatsby site from Netlify CMS 2 to Decap CMS 3:

1. uninstall gatsby-plugin-netlify-cms and netlify-cms-app
2. install gatsby-plugin-netlify-cms and decap-cms-app
3. replace the import `import CMS from "netlify-cms-app"` with `import CMS from "decap-cms-app"`
4. in gatsby config replace `resolve: "gatsby-plugin-netlify-cms"` with `resolve: "gatsby-plugin-decap-cms"`

An example of a migration can be found in [this PR on the Decap-Gatsby starter template](https://github.com/decaporg/gatsby-starter-decap-cms/commit/0b102651c68fd6fda064d30e032717cdcd74e9ce).

[Plugin on npmjs.com](https://www.npmjs.com/package/gatsby-plugin-decap-cms)

[Plugin on gatsbyjs.com](https://www.gatsbyjs.com/plugins/gatsby-plugin-decap-cms/)

---

_Copied from https://decapcms.org/blog/2024/02/new-gatsby-plugin-for-decap-is-now-available/_

**Comment by alexlibe95** [comment]
3. _replace the import `import CMS from "netlify-cms-app"`_ (small correction)

**Reply by martinjagodic** [reply]
thanks, fixed
